assets/_sass/components/_embeds.scss

```diff
@@ -1,6 +1,6 @@
 .embed-container {
   position: relative;
-  padding-bottom: 100%;
+  padding-bottom: 56.25%;
```

Comment:
Would there also need to be a `padding-top: 25px;` to allow for the YouTube chrome on the embedded player?